### ISSUE TITLE
Implicit strings concatenation should raise a violation when using commas

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#362]
# Rule request

## Thesis

We need to take care of this example:

```python
letters = ('a', 'b' 'c')
```

It is clear, that this tuple is just missing a comma. It is not a place for implicit sting concatenation. 

There's another example like:

```python
letter = (
     'Dear Sally!'
     'It has been a while since I heard from you '
     '...'
)
```

I thing that it is pretty fine. I use this kind of strings when there's a need in a long single line item.

## Reasoning

It can hide a future error.

**Comment by sobolevn** [comment]
This should be checked with tokens, not `ast`:

```
>>> import ast
>>> code = '"a" "b"'
>>> for i in ast.walk(ast.parse(code)):
...      print(i, vars(i))
...
<_ast.Module object at 0x107966710> {'body': [<_ast.Expr object at 0x10796c320>]}
<_ast.Expr object at 0x10796c320> {'value': <_ast.Str object at 0x10796c358>, 'lineno': 1, 'col_offset': 0}
<_ast.Str object at 0x10796c358> {'s': 'ab', 'lineno': 1, 'col_offset': 0}
```

**Comment by sobolevn** [comment]
Let's rephrase it to make this rule more general and easier to implement.
We need to forbid all implicit string concatenation on the same line.

That is just wrong:

```python
text = 'First' 'second'
tuple_with_parens = ('some' 'name', 'other' 'name)  # two issues
```

But, this is fine:

```python
long_text = (
    'First line'
    'Second line'
)
```

**Comment by sobolevn** [comment]
However, this does not protect us from these cases:

```python
this_is_a_tuple: Tuple[str, ...] = (
    'First item',
    'Second item'  # missing comma here
    'Last item',
)
```

**Comment by sobolevn** [comment]
So, maybe the rule should sound as: we only allow this form of multiline strings:

```python
this_is_a_string = (
     'First line'
    'Second line'
)
```

If we see any:
- commas
- any other tokens except newlines and a closing brackets

if we do not see:
- new lines between string tokens

We raise a violation.

**Comment by sobolevn** [comment]
Well, after some thoughts it became clear to me that the most reliable and correct way to handle this situation is to forbid implicit string concatenation at all.

Use `+` instead:


```python
this_is_string = (
    'First item' +
    'Second item' +
    'Last item'
)
```
